onboard/catkin_ws/src/controls/scripts/thruster_controls.py

```diff
@@ -67,7 +67,7 @@ def run(self):
                 self.t_allocs = np.clip(self.t_allocs, -1, 1)
 
                 i8_t_allocs = ThrusterSpeeds()
-                i8_t_allocs.speeds = (self.t_allocs * 60).astype(int)
+                i8_t_allocs.speeds = (self.t_allocs * 127).astype(int)
```

Comment:
Probably fine to keep master branch version, but keep in mind this can be changed

Reply:
Don't merge